src/dom-wrapper-error.ts

```diff
@@ -0,0 +1,58 @@
+import { FindComponentSelector, DOMWrapperAPI } from './types'
+
+interface Options {
+  selector: FindComponentSelector
+}
+
+export class DOMWrapperError implements DOMWrapperAPI {
+  private selector: FindComponentSelector
+  element: null
```

Comment:
Same for element, but I guess this is part of your followup PR.

Reply:
done